tests/python/gaia-ui-tests/gaiatest/mixins/gaia.py

```diff
@@ -21,6 +21,21 @@ class GaiaArguments(object):
          {'default': "undefined",
           'help': 'locale for the device, This value overrides the value from testvars.json file',
           }],
+        [['--capture'],
+         {'dest': 'capture',
+          'default': "whenfail",
```

Comment:
Yeah, I guess that, when I read the title of the bug again :neutral_face:  I left more explanation in Bug 1237069